packages/webapp/src/elections/components/ongoing-election-components/meeting-link/meeting-link.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from "react";
+import { Dayjs } from "dayjs";
+
+import {
+    delay,
+    encryptSecretForPublishing,
+    onError,
+    useEncryptedData,
+    useMemberGroupParticipants,
+    useUALAccount,
+    useVoteDataRow,
+    useZoomAccountJWT,
+} from "_app";
+import { generateZoomMeetingLink } from "_api/zoom-commons";
+import { setElectionMeeting } from "elections/transactions";
+import { calculateGroupId } from "elections/utils";
+import { ActiveStateConfigType, RoundStage } from "elections/interfaces";
+
+import MeetingButtons from "./meeting-buttons";
+import MeetingLinkModal from "./meeting-modal";
+
+export enum MeetingStep {
+    LinkZoomAccount,
+    CreateMeetingLink,
+    RetrieveMeetingLink,
+}
+
+interface RequestMeetingLinkProps {
+    roundIndex: number;
+    meetingStartTime: Dayjs;
+    meetingDurationMs: number;
+    electionConfig: ActiveStateConfigType;
+    stage: RoundStage;
+}
+
+/**
+ *
+ * For user flows and documentation, see ./request-election-meeting-link-button.md
```

Comment:
praise: this is great! I wonder if there is any plugin in vs code to handle and embed this ☺️ -- that would be even more awesome!

Reply:
That would be neat. I like the idea of referencing component documentation from the code.